### PR TITLE
Improved contract method decoded view

### PR DESCRIPTION
**Pull request by YegorSan**
This PR is related to #2537 issue 

## Motivation
The current state in contract method decoded view: too many white spaces, unnecessary nested tables

### Bug Fixes
Removed white spaces and unnecessary nested tables.

Screenshot

## Checklist for your PR

  - [x] I added an entry to `CHANGELOG.md` with this PR
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [ ] I checked whether I should update the docs and did so if necessary
  - [ ] If I added/changed/removed ENV var, I should update the list of env vars in https://github.com/poanetwork/blockscout/blob/master/docs/env-variables.md to reflect changes in the table here https://poanetwork.github.io/blockscout/#/env-variables?id=blockscout-env-variables. I've set `master` in the `Version` column.
  - [ ] If I add new indices into DB, I checked, that they don't redundant with PGHero or other tools

### CODE REVIEW

**Comment by YegorSan** [comment]
@vbaranov Hello Victor. I've connected to stg locally and will fix this issue soon.

**Comment by YegorSan** [comment]
@vbaranov Hello Victor. Could you please provide some additional steps to render verified contract locally. Actually I think this issue should be fixed with the #2746 PR, but anyway It's better to re-check everything locally.

File: apps/block_scout_web/assets/css/components/_tile.scss [hunk 1]
```diff
@@ -574,3 +574,7 @@ $cube-quantity: 5;
 .dark-theme-applied .dark-block-loader {
   background-color: #313355;
 }
+
+.table td, .table th {
+   vertical-align: middle;
+}
```
Comment (achempion) [inline apps/block_scout_web/assets/css/components/_tile.scss]: newline is missing at the end of the file

Comment (YegorSan) [reply apps/block_scout_web/assets/css/components/_tile.scss]: @achempion Hello. I don't think it's necessary for this application. If you check carefully css files you will find that it's not necessary rule, as this app is written in Elixir and using Phoenix framework. This convention is a holdover from C/C++ that many programmers still follow in more recent languages like Java out of habit.  Some C++ compilers prior to C++11 would emit a warning along the lines of "No newline at end of file", but as far as I can see all tests passed perfectly.

Comment (achempion) [reply apps/block_scout_web/assets/css/components/_tile.scss]: It's not about adding an extra newline at the end of a file, it's about not removing the newline that should be there. It is the standard of POSIX systems as well and it make tools such as `cat` handle files properly.

Additional info
- https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline
- https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Comment (YegorSan) [reply apps/block_scout_web/assets/css/components/_tile.scss]: @achempion If you check original file in master branch, you will find that there were not a newline in the end of that file - https://github.com/poanetwork/blockscout/blob/master/apps/block_scout_web/assets/css/components/_tile.scss 
So I've added a new required for this update class in the same way as it was before, and no newline should be there as you can see, and nothing were removed.

Comment (YegorSan) [reply apps/block_scout_web/assets/css/components/_tile.scss]: @achempion I've re-checked master branch locally , and didn't find an empty line in the end of that file, but I've added it by your request. Just updated the file.

**Pull request closed without merging**